src/components/policy/policy_regular/include/policy/policy_table/enums.h

```diff
@@ -52,9 +54,9 @@ const char* EnumToJsonString(Priority val);
 bool EnumFromJsonString(const std::string& literal, Priority* result);
 
 enum HmiLevel {
-  HL_BACKGROUND,
```

Comment:
Is there a reason we changed to ordering of the enum?

Reply:
Unit tests passing, WE should change either in unit tests, or in enums. I can see that ordering mistake is in enums, before my changes this unit test probably was not affected by it.

Reply:
@mrapitis return this changes back within enum ordering in unit tests.  66e229195